include/robot_localization/navsat_transform.h

```diff
@@ -210,6 +210,10 @@ class NavSatTransform
     //!
     bool use_local_cartesian_;
 
+    //! @brief Whether we want to have our UTM zone fixed, and not change it if we move outside of it
+    //!
+    bool force_user_utm_;
```

Comment:
`user`? It's typically robots :slightly_smiling_face: 

But it sounds like you don't force much. `reset_utm_`?

Reply:
It's similar to when a user calls the `/datum` service, from that point on a custom datum is enforced.
Now, when `/setUTMzone` is called, from that point on a utm datum is enforced.